Apps/Admin/Client/Pages/AnalyticsPage.razor

```diff
@@ -28,7 +28,17 @@
         <HgButton TopMargin="Breakpoint.Always" Disabled="@AnalyticsState.Value.IsLoading" EndIcon="@Icons.Material.Filled.Download" FullWidth="@true" Variant="Variant.Filled" Color="Color.Primary" @onclick="GetInactiveUsersData" data-testid="inactive-users-download-btn">Inactive Users</HgButton>
         <HgTextField T="int" Label="Days Inactive" Required="@true" @bind-Value="InactiveDays" Disabled="@AnalyticsState.Value.IsLoading" data-testid="days-inactive-input" />
     </MudItem>
-        <MudItem xs="6" lg="3">
-            <HgButton TopMargin="Breakpoint.Always" Disabled="@AnalyticsState.Value.IsLoading" EndIcon="@Icons.Material.Filled.Download" FullWidth="@true" Variant="Variant.Filled" Color="Color.Primary" @onclick="GetUserFeedbackData" data-testid="user-feedback-download-btn">User Feedback</HgButton>
-        </MudItem>
+    <MudItem xs="6" lg="3">
+        <HgButton TopMargin="Breakpoint.Always" Disabled="@AnalyticsState.Value.IsLoading" EndIcon="@Icons.Material.Filled.Download" FullWidth="@true" Variant="Variant.Filled" Color="Color.Primary" @onclick="GetUserFeedbackData" data-testid="user-feedback-download-btn">User Feedback</HgButton>
+    </MudItem>
+    <MudItem xs="6" lg="3">
+        <HgButton TopMargin="Breakpoint.Always" Disabled="@AnalyticsState.Value.IsLoading" EndIcon="@Icons.Material.Filled.Download" FullWidth="@true" Variant="Variant.Filled" Color="Color.Primary" @onclick="GetYearOfBirthCountsData" data-testid="user-feedback-download-btn">Year of Birth</HgButton>
```

Comment:
The `data-testid` should be updated so it is not identical to the `data-testid` on the User Feedback button.

Reply:
Oops I will update when doing the functional tests.